### PR TITLE
task solution

### PR DESCRIPTION
**Pull request by no4kar**
- [DEMO LINK](https://no4kar.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://no4kar.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+}
+
+.input--small {
+  font-family: Avenir, sans-serif;
+  font-weight: 900;
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
```
Comment (denys-danyliuk) [inline src/style.css:103]: 2'nd point in checklist

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.form-field-search--small::before {
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
```
Comment (denys-danyliuk) [inline src/style.css:58]: 2'nd point in checklist

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
```
Comment (denys-danyliuk) [inline src/style.css:5]: do you need this one? I believe that on a design there is only 400, and 900

File: src/index.html [hunk 4]
```diff
@@ -10,7 +10,35 @@
...
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="form-field-search form-field-search--big"
+    >
+      <input
+        type="text"
+        name="search-big"
```
Comment (denys-danyliuk) [inline src/index.html:23]: 6'th point in checklist

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (no4kar) [reply src/style.css:5]: I thought this is for this
### Requirements:
...
- default `font-weight` must be 300.

Comment (Andriy-Shymkiv) [reply src/style.css]: you did it in body styles already

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:6]: please remove it

**Review by Andriy-Shymkiv — APPROVED** [review]
remove this fonts and run test again. if that doesn't work, then leave it as it is

**Pull request closed without merging**